openapi.yaml

```diff
@@ -164,6 +184,9 @@ components:
           example: |
             gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf
             gitlab.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFSMqzJeV9rUzU4kWitGjeR4PWSa29SPqJ1fVkhtj3Hw9xjLVXVYrU9QlYWrOLXBpQ6KWjbjTDTdDkoohFzgbEY=
+        revision:
```

Comment:
We should probably move this out of the general `GitRepo` type and create a new type (e.g. `TenantGitRepo`) based on `GitRepo` adding the `revision` field there?